assets/sass/protocol/includes/mixins/_images.scss

```diff
@@ -23,18 +24,17 @@
     $slice: 2000;
     $index: 0;
     $loops: math.ceil(math.div(string.length($svg), $slice));
-
     @for $i from 1 through $loops {
         $chunk: string.slice($svg, $index, $index + $slice - 1);
 
         //   Encode
-        $chunk: str-replace($chunk, '"', "\'");
-        $chunk: str-replace($chunk, '%', '%25');
-        $chunk: str-replace($chunk, '#', '%23');
-        $chunk: str-replace($chunk, '{', '%7B');
-        $chunk: str-replace($chunk, '}', '%7D');
-        $chunk: str-replace($chunk, '<', '%3C');
-        $chunk: str-replace($chunk, '>', '%3E');
+        $chunk: functions.str-replace($chunk, '"', '\'');
+        $chunk: functions.str-replace($chunk, '%', '%25');
+        $chunk: functions.str-replace($chunk, '#', '%23');
+        $chunk: functions.str-replace($chunk, '{', '%7B');
+        $chunk: functions.str-replace($chunk, '}', '%7D');
+        $chunk: functions.str-replace($chunk, '<', '%3C');
+        $chunk: functions.str-replace($chunk, '>', '%3E');
```

Comment:
**Question(possible nit):** Did we need to bring in the `functions` file + namespace here? If I'm not mistaken, the `str-replace` would still be able to work without mentioning the file?

Reply:
Nice catch, after testing it out, I removed the str-replace that existed in 'images' since the one in functions is basically the same

Reply:
I still see the `functions.` in `images.scss`..?

Reply:
I still see the functions. in images.scss..?